pallets/parachain-staking/src/lib.rs

```diff
@@ -577,7 +586,7 @@ pub mod pallet {
 		Twox64Concat,
 		T::AccountId,
 		CollatorSnapshot<T::AccountId, BalanceOf<T>>,
-		ValueQuery,
+		OptionQuery,
```

Comment:
I don't think this would need a migration, but be careful about it!

You can still query the existence of a storage value when it's a `ValueQuery`, FYI: [contains_key](https://paritytech.github.io/substrate/master/frame_support/storage/trait.StorageDoubleMap.html#tymethod.contains_key)